File: frontend/scrape_devices.py
import urllib.request
import re

url = 'https://ufin.online/'
req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
html = urllib.request.urlopen(req).read().decode('utf-8')

# Find all device items
items = re.findall(r'<div class="elementor-widget-container">\s*<div class="card-item[^>]*>\s*<img[^>]+src="([^"]+)"[^>]*>\s*<h3>([^<]+)</h3>\s*<p>([^<]+)</p>', html)

for img, title, desc in items:
    print(f"Title: {title.strip()}")
    print(f"Image: {img.strip()}")
    print(f"Desc: {desc.strip()}")
    print("---")

File: frontend/scrape_devices.js
const cheerio = require('cheerio');
const fs = require('fs');

async function scrape() {
  const res = await fetch('https://ufin.online/');
  const html = await res.text();
  const $ = cheerio.load(html);
  
  const items = [];
  $('.swiper-slide').each((i, el) => {
    const title = $(el).find('h3').text().trim();
    const img = $(el).find('img').attr('src');
    const desc = $(el).find('p').text().trim();
    if (title) {
      items.push({ title, img, desc });
    }
  });
  console.log(JSON.stringify(items, null, 2));
}

scrape();
